qmetaobject/src/lib.rs

```diff
@@ -209,6 +209,16 @@ pub mod tablemodel;
 #[cfg(feature = "webengine")]
 pub mod webengine;
 
+/// Module intended for glob import.
+pub mod prelude {
+    pub use crate::{
+        lazy_static, qmetaobject_lazy_static, qml_register_type, qrc, qt_base_class, qt_method,
```

Comment:
qmetaobject_lazy_static shouldn't be necessary anymore with the changes i did yesterday.

Also, why is lazy_static in there?

Reply:
... oh. I was probably working on an oldish version then. Awesome!

Reply:
FYI, `lazy_static` is in there, because the `qmeta`-reexport actually doesn't 100% work. `qmetaobject;:*` always pulled `lazy_static` in "by accident" which made it work:

![image](https://user-images.githubusercontent.com/1263440/119142228-85c9bd00-ba46-11eb-899d-bbbbd72a604f.png)

I just tried without `qmetaobject_lazy_static`, which still gives me errors on `lazy_static` not being present.

Reply:
Found it, qmeta needs a $crate prefix in its body too :-)